src/OnboardingSPA/components/Sidebar/components/Customize/DesignColorsPanel/index.js

```diff
@@ -168,20 +223,29 @@ const DesignColorsPanel = forwardRef(
 				return;
 			}
 
-			colorPalettes[ selectedPalette ].primary = selectedColor.primary;
-			if ( colorPalettes[ selectedPalette ].secondary ) {
-				colorPalettes[ selectedPalette ].secondary =
+			if ( selectedPalette === 'custom' ) {
+				currentData.sitegen.homepages.data[ slug ].color.customColors =
```

Comment:
Can we use `currentData.sitegen.homepages.active` instead of `currentData.sitegen.homepages.data[slug]`, which would translate to the same? This applies to every single instance of `currentData.sitegen.homepages.data[slug]`.